terraform/google_gke/variables.tf

```diff
@@ -19,7 +19,7 @@ variable "disk_type" {
 
 variable "disk_size" {
   type = string
-  default = 50
+  default = 40
```

Comment:
This change was missed in the changelog.
Furthermore, why aren't there size units. My educated guess is these are GB or GiB but i am not sure. I would at least expect a comment if the variable refactoring turns out too complicated.

Reply:
I am not sure with the changelog as this repository contains various different infrastructure projects and the releases only represent releases for the OVF-image which is completely unrelated to the terraform modules :/

Changing the variable name would be a breaking change for our customers but i can add a comment.